config.lua.dist

```diff
@@ -80,6 +80,13 @@ freeDepotLimit = 2000
 premiumDepotLimit = 10000
 depotBoxes = 20
 
+-- Augment system
+-- NOTE: the following values are for all weapons and equipments that have type of "increase damage", "powerful impact" and "strong impact".
+-- To customize the percentage of a particular item, please add to the item "augments" section on items.xml
+augmentIncreasedDamageDefault = 5 * 100 -- 5%
+augmentPowerfulImpactDefault = 7 * 100 -- 7%
+augmentStrongImpactDefault = 10 * 100 -- 10%
```

Comment:
please remove spaces before commenting in the config.lua, you can put info into NOTE.
I recommend you change the config types to int (example: 5 to be 5%) and you'll do the conversion in your scripts, to avoid user bugs ..